src/SimMatrix.cpp

```diff
@@ -309,3 +309,35 @@ SimMatrix SimMatrix::DoSimStepReturnMatrix(const MatrixSetup& setup)
 	*this = locSimMatrix;
 	return locSimMatrix;
 }
+
+// only for cells not from edges (with margins)
+void SimMatrix::MyLifeNeighbors(SimMatrix& sMatrix)
+{
+    for (auto i = sMatrix.simMatrix.begin() + 1; i != sMatrix.simMatrix.end() - 1; ++i)
```

Comment:
You can actually name it a "row" or "column" instead of "i" and "j". That would make it more understandable and easier to read.